docs/tutorial/native-file-drag-drop.md

```diff
@@ -1,29 +1,44 @@
 # Native File Drag & Drop
 
+## Overview
+
 Certain kinds of applications that manipulate files might want to support
 the operating system's native file drag & drop feature. Dragging files into
 web content is common and supported by many websites. Electron additionally
 supports dragging files and content out from web content into the operating
 system's world.
 
-To implement this feature in your app, you need to call `webContents.startDrag(item)`
+To implement this feature in your app, you need to call
+[`webContents.startDrag(item)`](../api/web-contents.md#contentsstartdragitem)
 API in response to the `ondragstart` event.
 
-In your renderer process, handle the `ondragstart` event and forward the
-information to your main process.
+## Example
+
+Assuming you have a working Electron application from the
+[Quick Start Guide](quick-start.md), add the following lines to the
```

Comment:
Linking back to quick-start is a nice edition; someone landing on this page will be able to walk back to a starting point & be able to make running code.

I can imagine this preface becoming repetitive if we add it to more and more code samples, so I golfed it slightly:

```suggestion
Starting with a working application from the
[Quick Start Guide](quick-start.md), add the following lines to the
```